Extensions/popupLyrics.js

```diff
@@ -158,27 +188,41 @@ function PopupLyrics() {
 			}
 		}
 
+		/**
+		 * Search with PyNCM api.
+		 *
+		 * @param {Info} info
+		 * @returns {{
+		 * 	lyrics: [
+		 * 		{
+		 * 			startTime: Number,
+		 * 			text: string,
+		 * 		}
+		 * 	],
```

Comment:
Usually you don't need to add this `@returns` since your IDE or VS Code will determine it automatically. If you still want to annotate this, you should union `{ "error": string }` according to your return value.